modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -718,6 +718,57 @@ definition {
 			actionName = "Update Action Name");
 	}
 
+	@description = "LPS-156346 - Assert an action with an Expression using Groovy Script will resolve."
+	@priority = "3"
```

Comment:
I think the priority would be "5"

Reply:
Since this is a priority 5 test, we should add this to acceptance. @epark18 can you include the property to add the test to acceptance? You can forward this after you're done. Great job with the test btw